pom.xml

```diff
@@ -70,6 +70,13 @@
             <artifactId>sshd</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Without this, org.jenkins-ci.modules:sshd:2.6 pulls in the too-old 1.4. -->
```

Comment:
Could be dropped from this PR I think?

Reply:
Well without this, `mvn compile` fails, so the PR can't be validated.

```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.jenkins-ci.modules:ssh-cli-auth:1.4 paths to dependency are:
+-org.jenkins-ci.plugins:matrix-project:1.17-SNAPSHOT
  +-org.jenkins-ci.modules:sshd:2.6
    +-org.jenkins-ci.modules:ssh-cli-auth:1.4
and
+-org.jenkins-ci.plugins:matrix-project:1.17-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-war:2.243
    +-org.jenkins-ci.modules:ssh-cli-auth:1.8
```

Reply:
Still? You are no longer changing anything else in the POM in this PR, after I merged in #68.

Reply:
Never mind - had pulled master, but not the branch. With HEAD, it can indeed be dropped.